taichi/backends/opengl/opengl_api.cpp

```diff
@@ -12,7 +12,7 @@ namespace opengl {
 bool opengl_has_GL_NV_shader_atomic_float;
 
 #ifdef TI_WITH_OPENGL
-void glapi_set_uniform(GLuint loc, float value) {
+void glapi_set_uniform(unsigned loc, float value) {
```

Comment:
I actually feel like GLuint is more specific in this file since that indicates that these parameters are for the OpenGL API. Could you let me know the reason to use`unsigned` here? :-)

Reply:
I used to come to a solution which needs `GLuint` to occupy in the header, now it's not necessary, will change it back now.

Reply:
In that case, you may do `using GLuint = std::uint32_t` if OpenGL headers are not included.